board/ix-board.mk

```
@@ -4,7 +4,12 @@ $(2)_VERSION = ix-board
 $(2)_LICENSE = BSD-3-Clause
 $(2)_LICENSE_FILES = LICENSE
 $(2)_SITE_METHOD = local
-$(2)_SITE = $$(BR2_EXTERNAL_INFIX_PATH)/src/board/$(1)
+# Derive board path from board location: board/arch/vendor/model -> arch/vendor/model
+# Strip "board/" prefix from the board directory (passed as $(3))
+ifndef $(2)_BOARD_PATH
+$(2)_BOARD_PATH = $(3)
+endif
+$(2)_SITE = $$(BR2_EXTERNAL_INFIX_PATH)/board/$$($(2)_BOARD_PATH)
```

```suggestion
$(2)_SITE ?= $$(BR2_EXTERNAL_INFIX_PATH)/board/$(3)
```
And then if a board needs a custom path (e.g., because it is in a different external, then it can set `SITE` before calling the macro. Would that break anything?